cpp/src/community/louvain.cu

```diff
@@ -42,5 +79,17 @@ template std::pair<size_t, double> louvain(raft::handle_t const &,
                                            int32_t *,
                                            size_t,
                                            double);
+template std::pair<size_t, float> louvain(
+  raft::handle_t const &,
+  experimental::graph_view_t<int32_t, int32_t, float, false, false> const &,
```

Comment:
If instantiate for vertex_t,edge_t = int32_t, then we're still limited to work with fewer than 2B vertices/edges (and I found that I did this too in my PR838, so you may copied this :-(...), so I guess we need to fix this. For the new graph class, it's explicitly instantiated for (int32_t, init32_t), (int32_t, int64_t), and (int64_t, int64_t) combinations (https://github.com/rapidsai/cugraph/blob/branch-0.16/cpp/src/experimental/graph.cu#L510).

Reply:
This PR is intended to help bootstrap the python bindings integration work.  I agree that adding the int64_t variations will ultimately have to be done as well.  I'm sure I'll need it once I start testing MNMG Louvain on multiple GPUs, so my intention would be to add it to my implementation PR.

Reply:
Upon further reflection (and since I had to make an update anyway) I have added the additional explicit template instantiations.